types/d3-array/d3-array-tests.ts

```diff
@@ -341,99 +347,133 @@ stringyNumbersArray = ['0', '2', '3', '4', '7', '8'];
 dateArray = [new Date(2010, 1, 1), new Date(2011, 1, 1), new Date(2012, 1, 1), new Date(2013, 1, 1)];
 typedArray = Uint8Array.from(numbersArray);
 
-// scan() ----------------------------------------------------------------------
+// least() ----------------------------------------------------------------------
```

Comment:
was scan fully removed? I guess it makes sense to remove the tests as it is deprectaed though

Reply:
The documentation for it was removed in https://github.com/d3/d3-array/commit/09b8ff21742b365a7090e728120640fb45464637 (before the `2.3.0` release), so I assumed it was just an oversight that it wasn't fully removed in `2.0.0`.